Gemfile.lock

```diff
@@ -209,60 +209,60 @@ GEM
       sprockets-rails
       tilt
     semantic_range (2.3.0)
-    spring (2.1.0)
+    spring (2.1.1)
     spring-watcher-listen (2.0.1)
       listen (>= 2.7, < 4.0)
       spring (>= 1.2, < 3.0)
     sprockets (4.0.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
-    sprockets-rails (3.2.1)
+    sprockets-rails (3.2.2)
       actionpack (>= 4.0)
       activesupport (>= 4.0)
       sprockets (>= 3.0.0)
     thor (1.0.1)
     thread_safe (0.3.6)
     tilt (2.0.10)
-    tinymce-rails (5.4.1)
+    tinymce-rails (5.6.1)
       railties (>= 3.1.1)
-    trestle (0.9.3)
+    trestle (0.9.4)
```

Comment:
### [trestle](https://www.trestle.io)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 0.9.3 → 0.9.4

[_~~change-log~~, [source-code](https://github.com/TrestleAdmin/trestle)_]





<details>
<summary>Commits</summary>

A change of **56** commits. See the full changes on [the compare page](https://github.com/TrestleAdmin/trestle/compare/v0.9.3...v0.9.4).

These are the first 10 commits:
- (26334fe) [Use Admin#translate to generate default scope label](https://github.com/TrestleAdmin/trestle/commit/26334fe3d987ba3336113566d92b2793f5c47154)
- (31caf13) [Set up admin reloader within after_initialize block](https://github.com/TrestleAdmin/trestle/commit/31caf131cac48a8b177aa2e8894d0f019355a0f7)
- (245942e) [Move reloader initialization to Reloader#install method](https://github.com/TrestleAdmin/trestle/commit/245942e95da44b3d5c5395cae010f61368e67a14)
- (bdb9d1c) [Don't install reloader when app.config.eager_load is set](https://github.com/TrestleAdmin/trestle/commit/bdb9d1c97cb8af3808898755b519e9cfea5d015d)
- (9efa928) [Update Bootstrap to 4.5.0 and jQuery to 3.5.1](https://github.com/TrestleAdmin/trestle/commit/9efa928719a61b290904e932accb14bc7eb5b8ac)
- (79e003b) [Update rails-ujs to 6.0.3](https://github.com/TrestleAdmin/trestle/commit/79e003bee023c0e8308dd251e9cef29b29442a14)
- (fce849b) [Update JS pipeline dependencies](https://github.com/TrestleAdmin/trestle/commit/fce849bf7b221d695c98bb226e0098bea89ebe9e)
- (9d1d7f0) [Make Reloader track all watchable files and dirs](https://github.com/TrestleAdmin/trestle/commit/9d1d7f07aca3dcc70ae574e2e7a0d9d6038872c4)
- (a457782) [Implement preliminary support for AJAX pagination](https://github.com/TrestleAdmin/trestle/commit/a4577826ba37c926f03a425a334087c07dbafd00)
- (99d5902) [Update JS pipeline dependencies](https://github.com/TrestleAdmin/trestle/commit/99d59029e2e5f9fcb6fcb65ba52ecf3b9b8d1a19)


</details>